.github/workflows/cd.yaml

```diff
@@ -0,0 +1,59 @@
+# Note: additional setup is required, see https://www.jenkins.io/redirect/continuous-delivery-of-plugins
```

Comment:
is it really related to a cleanup?

Reply:
mostly the modernization part ;)

Reply:
Maybe you could replace Jenkinsfile by a GitHub action as CI as well? 😜🤣

Reply:
please, no :sob:

Reply:
That's ok I can make the PR for that